git/week1/dogage.js

```diff
@@ -0,0 +1,17 @@
+let dogYearOfBirth=2035;
+let dogYearFuture =2045;
+let dogYear = dogYearFuture-dogYearOfBirth;
+let dogandhumanyear=" dog ";
+let dogage;
```

Comment:
Variable names are conventionally created as `camelCased` to make them more readable, especially if they are composed of multiple words. How would these variables look like to meet this convention?